src/main/java/com/sprint/mission/discodeit/service/basic/BasicMessageService.java

```diff
@@ -1,89 +1,137 @@
 package com.sprint.mission.discodeit.service.basic;
 
-import com.sprint.mission.discodeit.domain.Channel;
+import com.sprint.mission.discodeit.domain.BinaryContent;
 import com.sprint.mission.discodeit.domain.Message;
-import com.sprint.mission.discodeit.domain.User;
-import com.sprint.mission.discodeit.error.ErrorCode;
+import com.sprint.mission.discodeit.domain.UserStatus;
+import com.sprint.mission.discodeit.dto.message.CreateMessageRequest;
+import com.sprint.mission.discodeit.dto.message.UpdateMessageRequest;
+import com.sprint.mission.discodeit.dto.userstatus.UpdateUserStatusRequest;
+import com.sprint.mission.discodeit.exception.ErrorCode;
 import com.sprint.mission.discodeit.exception.ServiceException;
+import com.sprint.mission.discodeit.repository.BinaryContentRepository;
 import com.sprint.mission.discodeit.repository.ChannelRepository;
 import com.sprint.mission.discodeit.repository.MessageRepository;
 import com.sprint.mission.discodeit.repository.UserRepository;
 import com.sprint.mission.discodeit.service.MessageService;
+import com.sprint.mission.discodeit.service.UserStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
 
+import java.time.Instant;
 import java.util.List;
+import java.util.Optional;
 import java.util.UUID;
+import java.util.stream.Collectors;
 
+@Service
+@RequiredArgsConstructor
 public class BasicMessageService implements MessageService {
     private final MessageRepository messageRepository;
     private final UserRepository userRepository;
     private final ChannelRepository channelRepository;
-
-    public BasicMessageService(MessageRepository messageRepository, UserRepository userRepository, ChannelRepository channelRepository) {
-        this.userRepository = userRepository;
-        this.messageRepository = messageRepository;
-        this.channelRepository = channelRepository;
-    }
+    private final BinaryContentRepository binaryContentRepository;
+    private final UserStatusService userStatusService;
 
     @Override
-    public Message create(String content, User writer, Channel channel) {
-        if (content.isEmpty()) {
+    public Message create(CreateMessageRequest request) {
+        if (request.content().isEmpty()) {
             throw new ServiceException(ErrorCode.EMPTY_CONTENT);
         }
 
-        existUser(writer);
-        existChannel(channel);
-        Message message = new Message(content, writer, channel);
+        // 작성자와 채널에 대한 검증
+        validUser(request.writerID());
+        validChannel(request.channelID());
+
+        // User가 메시지를 만들면 UserStatus 업데이트하기
+        UserStatus userStatus = userStatusService.findByUserId(request.writerID()).orElseThrow(() -> new ServiceException(ErrorCode.CANNOT_FOUND_USERSTATUS));
+        UpdateUserStatusRequest updateRequest = new UpdateUserStatusRequest(userStatus.getId(), Instant.now());
+        userStatusService.update(updateRequest);
```

Comment:
```suggestion
        UpdateUserStatusRequest updateRequest = new UpdateUserStatusRequest(request.writerID(), Instant.now());
        userStatusService.update(updateRequest);
```
이미 update 내부에서도 조회가 되고 있는데 사용자 id정도만 넘기면 괜찮을듯 해요~